app/services/testdata/InProgressPersonalDetailsStatusGenerator.scala

```diff
@@ -61,14 +61,23 @@ trait InProgressPersonalDetailsStatusGenerator extends ConstructiveGenerator {
       )
     }
 
+    def getFastPassDetails(candidateInformation: DataGenerationResponse) = {
+      if (generatorConfig.isCivilServant.contains(true)) {
+        FastPassDetails(true, Some(FastPassType.CivilServant), None, Some(true), Some("123ABC"))
```

Comment:
Shall we change the last two parameters Some(true), Some("123ABC") to None, None because candidate who is a civil servant do not fill the fastPassReceived and fastPassCertificate number